src/Resources/config/services.yml

```diff
@@ -1,10 +1,11 @@
 services:
-  shopsys.form_types.multidomain:
-    class: Shopsys\FormTypesBundle\MultidomainType
-    tags:
-      - { name: form.type }
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
 
-  shopsys.form_types.yes_no:
-    class: Shopsys\FormTypesBundle\YesNoType
-    tags:
-      - { name: form.type }
+  Shopsys\FormTypesBundle\:
+    resource: '../../*'
+
+  Shopsys\FormTypesBundle\Domain\DomainIdsProviderInterface:
```

Comment:
this should be in services.yml in shopsys framework

Reply:
fixed